spec/models/picture_spec.rb

```diff
@@ -9,6 +9,7 @@
   let(:picture2) { Ckeditor::Picture.create(data: file_fixture('image.png').open) }
   let(:picture3) { Ckeditor::Picture.create(data: file_fixture('image.png').open) }
   let(:picture4) { Ckeditor::Picture.create(data: file_fixture('image.png').open) }
+  let(:file)     { Ckeditor::Picture.create(data: file_fixture('diploma.pdf').open) }
```

Comment:
are you using this in the spec?

maybe it's in here so you can test the attachment type (not an image), but you don't finish doing it?